grafana/dashboards/health.json

```diff
@@ -16,16 +16,305 @@
   "gnetId": null,
   "graphTooltip": 1,
   "id": null,
-  "iteration": 1524697499292,
+  "iteration": 1529950676425,
   "links": [],
   "panels": [
+    {
+      "content": "<div class=\"text-center dashboard-header\">\n  <span>Data-Plane Telemetry</span>\n</div>",
+      "gridPos": {
+        "h": 2.2,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 400,
+      "links": [],
+      "mode": "html",
+      "title": "",
+      "transparent": true,
+      "type": "text"
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "prometheus",
+      "fill": 0,
+      "gridPos": {
+        "h": 7,
+        "w": 8,
+        "x": 0,
+        "y": 2.2
+      },
+      "id": 397,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": false,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 2,
+      "links": [],
+      "nullPointMode": "null",
+      "percentage": false,
+      "pointradius": 5,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "process_virtual_memory_bytes{conduit_io_control_plane_ns=\"$namespace\", job=\"conduit-proxy\"}",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "{{namespace}}/{{pod}}/virtual",
+          "refId": "A"
+        },
+        {
+          "expr": "process_resident_memory_bytes{conduit_io_control_plane_ns=\"$namespace\", job=\"conduit-proxy\"}",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "{{namespace}}/{{pod}}/resident",
+          "refId": "B"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "MEMORY USAGE",
+      "tooltip": {
+        "shared": true,
```

Comment:
Since these graphs have the potential to have many timeseries, it might be useful to change the tooltip to only display the series that is being hovered on. Right now it looks like this:

![image](https://user-images.githubusercontent.com/9226/41943224-7dbb80b8-7957-11e8-9e4c-34b08ba566c8.png)

But if you change the hover tooltip mode to "Single":

![image](https://user-images.githubusercontent.com/9226/41943247-91967ffc-7957-11e8-807c-87c10e3f1c45.png)

Then it will look like this:

![image](https://user-images.githubusercontent.com/9226/41943271-a4317856-7957-11e8-920d-23962949e25c.png)

I'd do that for all 3 graphs.